Lib/test/libregrtest/main.py

```diff
@@ -432,14 +433,26 @@ def display_header(self):
         print("== encodings: locale=%s, FS=%s"
               % (locale.getpreferredencoding(False),
                  sys.getfilesystemencoding()))
+        rl_version_cmd = """if 1:
         try:
             import readline
         except ImportError:
             readline = None
         if readline is not None:
-            print(f"== readline version: {readline._READLINE_VERSION:#06x}")
+            print(f"{readline._READLINE_VERSION:#06x}")
             is_editline = readline.__doc__ and "libedit" in readline.__doc__
-            print("== readline implementation:", "editline" if is_editline else "GNU readline")
+            print("editline" if is_editline else "GNU readline")
+        """
```

Comment:
It is very hard to read this code like this. The other program that you run should be indented differently from the other code.

Also the code to return the info should be in separate function, and here we can just print the values.